packages/one-app-bundler/utils/getConfigOptions.js

```diff
@@ -16,26 +16,38 @@ const readPkgUp = require('read-pkg-up');
 const get = require('lodash/get');
 const commonConfig = require('../webpack/webpack.common');
 
-function validateOptions(options) {
+function validateExternals(options) {
   if (options.requiredExternals && options.providedExternals) {
     throw new Error('@americanexpress/one-app-bundler: Modules cannot configure both requiredExternals and providedExternals. See README for details.');
   }
+}
 
-  if (options.requiredExternals || options.providedExternals) {
-    const intersection = Object.keys(commonConfig.externals)
-      .filter((externalName) =>
-        // eslint-disable-next-line implicit-arrow-linebreak
-        (options.requiredExternals || options.providedExternals).includes(externalName)
-      );
-    if (intersection.length > 0) {
-      throw new Error(`@americanexpress/one-app-bundler: Attempted to bundle ${intersection.join(', ')}, but modules cannot provide externals that One App includes.`);
+function validateOptions(options) {
+  validateExternals(options);
+  const allowedBundlerOptions = ['providedExternals', 'requiredExternals', 'webpackConfigPath', 'appCompatibility', 'purgecss', 'webpackClientConfigPath', 'webpackServerConfigPath'];
+  const allowedOptions = Object.keys(options).filter(
```

Comment:
what about using [every](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every) and changing the name to `optionsAreValid`

then below you can do 
```js
if (!optionsAreValid) {
     throw new TypeError('@americanexpress/one-app-bundler: Wrong syntax used for bundler options');
  }
```